android/src/main/java/co/il/nester/android/react/streetview/NSTStreetView.java

```diff
@@ -83,9 +85,12 @@ public void onStreetViewPanoramaCameraChange(StreetViewPanoramaCamera streetView
         panorama.setOnStreetViewPanoramaChangeListener(new StreetViewPanorama.OnStreetViewPanoramaChangeListener() {
             @Override
             public void onStreetViewPanoramaChange(StreetViewPanoramaLocation streetViewPanoramaLocation) {
-                if (streetViewPanoramaLocation != null && streetViewPanoramaLocation.links != null ) {
```

Comment:
I wasnt sure why its looking for links but it was preventing the link from firing when otherwise a change was made. I can add an extra if statement if there is something special we want to handle for this case.